driver-core/src/test/unit/com/mongodb/internal/connection/DefaultServerSpecification.groovy

```diff
@@ -334,7 +336,7 @@ class DefaultServerSpecification extends Specification {
         when:
         def futureResultCallback = new FutureResultCallback()
         testConnection.insertAsync(new MongoNamespace('test', 'test'), true, new InsertRequest(new BsonDocument()),
-                futureResultCallback);
+                null, futureResultCallback);
```

Comment:
Throughout this test class either `null` or `IgnorableRequestContext.INSTANCE` is passed in different places. My understanding is that we should always pass `IgnorableRequestContext.INSTANCE`.